src/index.ts

```diff
@@ -68,11 +67,19 @@ app.get(`/:id`, (request, response) => {
 app.post(`/`, (request, response) => {
     // Check if it is a valid url
     const { url } = request.body;
-    if (!url || !config.urlRegex.test(url)) {
+    
+    try {
+        url = new URL(string);
```

Comment:
You're overwriting the `url` variable here. Just `new URL(string);` should be fine too, as that should throw an error when an invalid URL is provided too (not sure, just assuming this).

Reply:
Yes, that should work.